GoMFT Update Fails

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

GoMFT

### 📂 What was the exact command used to execute the script?

update

### 📝 Provide a clear and concise description of the issue.

GoMFT update fails

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 🔄 Steps to reproduce the issue.

Run update within the container - get below error

### ❌ Paste the full error output (if available).

```
   ______      __  _______________
  / ____/___  /  |/  / ____/_  __/
 / / __/ __ \/ /|_/ / /_    / /   
/ /_/ / /_/ / /  / / __/   / /    
\____/\____/_/  /_/_/     /_/     
                                  

Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
build-essential is already the newest version (12.9).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
  ✔️   Stopped GoMFT
⠙ Updating GoMFT to 0.2.1(!) templ version check: generator v0.3.857 is newer than templ version v0.3.833 found in go.mod file, consider running `go get -u github.com/a-h/templ` to upgrade
⠼ Updating GoMFT to 0.2.1(✓) Complete [ updates=88 duration=402.469733ms ]
⠏ Updating GoMFT to 0.2.1# github.com/starfleetcptn/gomft/components
components/admin_tools_templ.go:17:6: BackupFile redeclared in this block
        components/admin_database_templ.go:17:6: other declaration of BackupFile
components/admin_tools_templ.go:854:76: backup.ModTime undefined (type BackupFile has no field or method ModTime)

[ERROR] in line 53: exit code 0: while executing command $STD go build -o gomft
```

### 🖼️ Additional context (optional).

running 'update' or the full command is the same.

Unfortunately not working but a different error at least

`   ______      __  _______________
  / ____/___  /  |/  / ____/_  __/
 / / __/ __ \/ /|_/ / /_    / /   
/ /_/ / /_/ / /  / / __/   / /    
\____/\____/_/  /_/_/     /_/     
                                  

Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
build-essential is already the newest version (12.9).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
  ✔️   Stopped GoMFT
⠴ Updating GoMFT to 0.2.3
[ERROR] in line 44: exit code 0: while executing command wget -q "https://github.com/StarFleetCPTN/GoMFT/archive/refs/tags/v${RELEASE}.tar.gz" -O $temp_file`

--- a/ct/gomft.sh
+++ b/ct/gomft.sh
@@ -28,7 +28,7 @@ function update_script() {
     msg_error "No ${APP} Installation Found!"
     exit
   fi
-  RELEASE=$(curl -s https://api.github.com/repos/StarFleetCPTN/GoMFT/releases/latest | grep "tag_name" | awk '{print substr($2, 4, length($2)-5) }')
+  RELEASE=$(curl -s https://api.github.com/repos/StarFleetCPTN/GoMFT/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
   if ! dpkg -l | grep -q "^ii.*build-essential"; then
     $STD apt-get install -y build-essential
   fi
@@ -41,7 +41,7 @@ function update_script() {
     msg_info "Updating $APP to ${RELEASE}"
     rm -f /opt/gomft/gomft
     temp_file=$(mktemp)
-    wget -q "https://github.com/StarFleetCPTN/GoMFT/archive/refs/tags/v.${RELEASE}.tar.gz" -O $temp_file
+    curl -fsSL "https://github.com/StarFleetCPTN/GoMFT/archive/refs/tags/v.${RELEASE}.tar.gz" -o $temp_file
     tar -xzf $temp_file
     cp -rf GoMFT-v.${RELEASE}/* /opt/gomft
     cd /opt/gomft

--- a/ct/gomft.sh
+++ b/ct/gomft.sh
@@ -28,7 +28,7 @@ function update_script() {
     msg_error "No ${APP} Installation Found!"
     exit
   fi
-  RELEASE=$(curl -s https://api.github.com/repos/StarFleetCPTN/GoMFT/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+  RELEASE=$(curl -fsSL https://api.github.com/repos/StarFleetCPTN/GoMFT/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
   if ! dpkg -l | grep -q "^ii.*build-essential"; then
     $STD apt-get install -y build-essential
   fi

--- a/ct/gomft.sh
+++ b/ct/gomft.sh
@@ -41,9 +41,9 @@ function update_script() {
     msg_info "Updating $APP to ${RELEASE}"
     rm -f /opt/gomft/gomft
     temp_file=$(mktemp)
-    wget -q "https://github.com/StarFleetCPTN/GoMFT/archive/refs/tags/v${RELEASE}.tar.gz" -O $temp_file
+    wget -q "https://github.com/StarFleetCPTN/GoMFT/archive/refs/tags/v.${RELEASE}.tar.gz" -O $temp_file
     tar -xzf $temp_file
-    cp -rf GoMFT-${RELEASE}/* /opt/gomft
+    cp -rf GoMFT-v.${RELEASE}/* /opt/gomft
     cd /opt/gomft
     $STD go mod download
     $STD go install github.com/a-h/templ/cmd/templ@latest

--- a/install/gomft-install.sh
+++ b/install/gomft-install.sh
@@ -34,7 +34,7 @@ msg_ok "Setup Golang"
 
 msg_info "Setup ${APPLICATION} (Patience)"
 temp_file=$(mktemp)
-RELEASE=$(curl -s https://api.github.com/repos/StarFleetCPTN/GoMFT/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+RELEASE=$(curl -fsSL https://api.github.com/repos/StarFleetCPTN/GoMFT/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
 curl -fsSL "https://github.com/StarFleetCPTN/GoMFT/archive/refs/tags/v.${RELEASE}.tar.gz" -o $temp_file
 tar -xzf $temp_file
 mv GoMFT-v.${RELEASE}/ /opt/gomft

--- a/install/gomft-install.sh
+++ b/install/gomft-install.sh
@@ -34,8 +34,8 @@ msg_ok "Setup Golang"
 
 msg_info "Setup ${APPLICATION} (Patience)"
 temp_file=$(mktemp)
-RELEASE=$(curl -s https://api.github.com/repos/StarFleetCPTN/GoMFT/releases/latest | grep "tag_name" | awk '{print substr($2, 4, length($2)-5) }')
-wget -q "https://github.com/StarFleetCPTN/GoMFT/archive/refs/tags/v.${RELEASE}.tar.gz" -O $temp_file
+RELEASE=$(curl -s https://api.github.com/repos/StarFleetCPTN/GoMFT/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+culr -fsSL "https://github.com/StarFleetCPTN/GoMFT/archive/refs/tags/v.${RELEASE}.tar.gz" -o $temp_file
 tar -xzf $temp_file
 mv GoMFT-v.${RELEASE}/ /opt/gomft
 cd /opt/gomft

--- a/install/gomft-install.sh
+++ b/install/gomft-install.sh
@@ -35,9 +35,9 @@ msg_ok "Setup Golang"
 msg_info "Setup ${APPLICATION} (Patience)"
 temp_file=$(mktemp)
 RELEASE=$(curl -s https://api.github.com/repos/StarFleetCPTN/GoMFT/releases/latest | grep "tag_name" | awk '{print substr($2, 4, length($2)-5) }')
-wget -q "https://github.com/StarFleetCPTN/GoMFT/archive/refs/tags/v${RELEASE}.tar.gz" -O $temp_file
+wget -q "https://github.com/StarFleetCPTN/GoMFT/archive/refs/tags/v.${RELEASE}.tar.gz" -O $temp_file
 tar -xzf $temp_file
-mv GoMFT-${RELEASE}/ /opt/gomft
+mv GoMFT-v.${RELEASE}/ /opt/gomft
 cd /opt/gomft
 $STD go mod download
 $STD go install github.com/a-h/templ/cmd/templ@latest

--- a/install/gomft-install.sh
+++ b/install/gomft-install.sh
@@ -35,7 +35,7 @@ msg_ok "Setup Golang"
 msg_info "Setup ${APPLICATION} (Patience)"
 temp_file=$(mktemp)
 RELEASE=$(curl -s https://api.github.com/repos/StarFleetCPTN/GoMFT/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
-culr -fsSL "https://github.com/StarFleetCPTN/GoMFT/archive/refs/tags/v.${RELEASE}.tar.gz" -o $temp_file
+curl -fsSL "https://github.com/StarFleetCPTN/GoMFT/archive/refs/tags/v.${RELEASE}.tar.gz" -o $temp_file
 tar -xzf $temp_file
 mv GoMFT-v.${RELEASE}/ /opt/gomft
 cd /opt/gomft

